Formula/zoxide.rb

```diff
@@ -0,0 +1,16 @@
+class Zoxide < Formula
+  desc "Shell extension to navigate your filesystem faster"
+  homepage "https://github.com/ajeetdsouza/zoxide"
+  url "https://github.com/ajeetdsouza/zoxide/archive/v0.4.1.tar.gz"
+  sha256 "bf3bed2d384e2f137c1d4cf56a97fb7e925abd69823ce41f1552c76e183466c8"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+  end
+
+  test do
+    assert_equal "/", shell_output("#{bin}/zoxide query /").strip
```

Comment:
We're going to need to check for something more than just the slash

Reply:
@SMillerDev I've changed the tests to create a new DB and add `/` to it, after which it checks if it got added successfully. I think this should be a sufficient test.

Reply:
What if it fails and returns a stacktrace with paths? That will also match this regex. Is there no way to check for some more output?

Reply:
Could you clarify what you mean by regex? Won't `assert_eq` provide a direct equality check?

Reply:
IMO, adding more directories won't improve the quality of the test much, since after that it will rely on matching logic which is always consistent across platforms. If the database can be read from and written into, I think it would be safe to say that zoxide is running correctly.